### PR TITLE
fix: use electron-log in main/core.js

### PR DESCRIPTION
**Pull request by bajtos**
Print the logs to the application's log file instead of the console.

I discovered this problem while trying to debug https://github.com/filecoin-station/desktop/issues/1659

### CODE REVIEW

File: main/core.js [hunk 1]
```diff
@@ -189,14 +192,14 @@ async function maybeMigrateFiles () {
   } catch {
     return
   }
-  console.log(
+  log.info(format(
     'Migrating files from %s to %s',
     oldSaturnDir,
     newSaturnDir
-  )
+  ))
```
Comment (bajtos) [inline main/core.js:199]: I think we can remove `maybeMigrateFiles` now that Station no longer ships Saturn L2 Node. WDYT, @juliangruber? I can open a follow-up pull request.

Comment (juliangruber) [reply main/core.js:199]: yes please! good spot

File: main/core.js [hunk 2]
```diff
@@ -87,7 +90,7 @@ async function start (ctx) {
     },
     stdio: ['pipe', 'pipe', 'pipe', 'ipc']
   })
-  console.log('Core pid', childProcess.pid)
+  log.info(format('Core pid: %s', childProcess.pid))
```
Comment (juliangruber) [inline main/core.js:93]:
```suggestion
  log.info(format('Core pid:', childProcess.pid))
```

for consistency

**Review by juliangruber — APPROVED** [review]

File: main/core.js (hunk 1 above)
Comment (bajtos) [reply main/core.js:199]: https://github.com/filecoin-station/desktop/pull/1668

**Pull request merged**